### PR TITLE
Support invoking the C++ evm2wasm directly

### PR DESCRIPTION
**Pull request by axic**

### CODE REVIEW

File: src/hera.cpp [hunk 1]
```diff
@@ -50,6 +52,8 @@ using namespace HeraVM;
 enum hera_evm_mode {
   EVM_FALLBACK,
   EVM2WASM,
```
Comment (jakelang) [inline src/hera.cpp]: This option is not needed anymore?

Comment (jakelang) [inline src/hera.cpp]: Maybe rename this to EVM2WASM_CONTRACT for clarity

Comment (axic) [reply src/hera.cpp]: Do you want to create a PR for that?

Comment (jakelang) [reply src/hera.cpp]: @axic #235

**Pull request merged**